etc/config/vb.defaults.properties

```diff
@@ -52,7 +52,7 @@ compiler.dotnettrunkvbcoreclr.exe=/opt/compiler-explorer/dotnet-trunk/.dotnet/do
 compiler.dotnettrunkvbcoreclr.name=.NET CoreCLR (main)
 compiler.dotnettrunkvbcoreclr.clrDir=/opt/compiler-explorer/dotnet-trunk
 compiler.dotnettrunkvbcoreclr.buildConfig=Release
-compiler.dotnettrunkvbcoreclr.langVersion=preview
```

Comment:
VB doesn't support `preview`, previously MSBuild just ignored this option. Changed to `latest` to avoid compilation errors.

Reply:
Thanks for the explanaation